examples/pipelines/kritis-resources.yaml

```diff
@@ -10,26 +10,13 @@ spec:
     value: https://github.com/grafeas/kritis
   - name: revision
     value: master
----    
-apiVersion: pipeline.knative.dev/v1alpha1
-kind: PipelineResource
-metadata:
-  name: kritis-resources-test-git
-  namespace: default
-spec:    
-  type: git
-  params:
-  - name: revision
-    value: master
-  - name: url
-    value: https://github.com/grafeas/kritis-test  
----    
+---
```

Comment:
There are use cases when having tests in a separate repo is valid, for example having acceptance test that make sure products from different repos integrate and deploy properly. Anyways doesn't have to be part of our example, but it was good way to show using multiple git repos in the same task

Reply:
I’m okay with your change :)